stable/rails/templates/ingress-public.yaml

```diff
@@ -1,10 +1,10 @@
-{{ if (and .Values.webserver.enabled .Values.ingress.enabled) }}
+{{ if (and .Values.webserver.enabled .Values.ingress_public.enabled) }}
```

Comment:
How do we specify which ingress controller are we using (public or private)?

Reply:
add values - type: public/private